File: app/src/main/java/com/example/kineduexample/entities/DataActivities.java
package com.example.kineduexample.entities;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DataActivities {
    @SerializedName("id")
    private Integer id;

    @SerializedName("name")
    private String name;

    @SerializedName("type")
    private String type;

    @SerializedName("activities")
    private List<Activities> activities;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public List<Activities> getActivities() {
        return activities;
    }

    public void setActivities(List<Activities> activities) {
        this.activities = activities;
    }

    @Override
    public String toString() {
        return "DataActivities{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", type='" + type + '\'' +
                ", activities=" + activities +
                '}';
    }
}

File: app/src/main/java/com/example/kineduexample/data/network/model/DataActivities.kt
package com.example.kineduexample.data.network.model

import com.google.gson.annotations.SerializedName

class DataActivities {
    @SerializedName("id")
    var id: Int? = null

    @SerializedName("name")
    var name: String? = null

    @SerializedName("type")
    var type: String? = null

    @SerializedName("activities")
    var activities: List<Activities>? = null

    override fun toString(): String {
        return "DataActivities{" +
                "id=" + id +
                ", name='" + name + '\''.toString() +
                ", type='" + type + '\''.toString() +
                ", activities=" + activities +
                '}'.toString()
    }
}
